prompt_toolkit/key_binding/key_processor.py

```diff
@@ -387,7 +398,7 @@ def wait():
             " Wait for timeout. "
             time.sleep(self.timeout)
 
-            if counter == self._keys_pressed:
```

Comment:
@jonathanslenders I think that it should check `key_buffer` instead of `input queue`, right?